### PR TITLE
[OSDOCS-3997] OCP content port to ROSA and OSD: Support

### PR DESCRIPTION
**Pull request by mburke5678**
This PR ports the Support book from OCP to the OSD and ROSA books using single-sourcing.

This PR is to add the Support book to OSD and ROSA and to move/remove the existing ROSA/OSD Support assemblies to the Support book, as appropriate. There is little or no new text. All the changes are adding assemblies, adding ifdefs, updating links, etc. 

Version(s):

Merge to `enterprise-4.13`, cherrypick to `main`, then CP that PR to `enterprise-4.14`.

Previews:
[OCP](https://63305--docspreview.netlify.app/openshift-enterprise/latest/support/)
[ROSA](https://63305--docspreview.netlify.app/openshift-rosa/latest/support/)
[OSD](https://63305--docspreview.netlify.app/openshift-dedicated/latest/support/)

Issue: https://issues.redhat.com/browse/OSDOCS-3997

### CODE REVIEW

**Comment by mburke5678** [comment]
@xueli181114  QE to review this change.

File: support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc [hunk 1]
```diff
@@ -13,11 +13,7 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 You may choose to opt out of reporting health and usage data for your cluster.
 endif::[]
 
-ifdef::openshift-dedicated[]
-On {product-title}, remote health reporting is always enabled. You cannot opt out.
-
-// TODO: Is there even a reason to include this assembly for OSD? Could you remove it, and just mention in the other assemblies that you can't opt out?
-endif::[]
+// moved OSD note on not able to opt out to about telemetery
```
Comment (mburke5678) [inline support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc:16]: Moved to: https://github.com/openshift/openshift-docs/pull/63305/files#diff-a380b4a4a3b7cdf4ef40cbe2d4504fba42212a1ba48b3ec9a700eeec15f0ffa0R37

File: support/troubleshooting/troubleshooting-s2i.adoc
Comment (kcarmichael08) [inline support/troubleshooting/troubleshooting-s2i.adoc:1]: This assembly doesn't seem to be in the OSD & ROSA books, but I guess having the ifndef to not show the additional resources section doesn't hurt anything.

**Review by kcarmichael08 — COMMENTED** [review]
checked files in troubleshooting folder, lgtm

**Review by kcarmichael08 — COMMENTED** [review]
reviewed files in remote_health_monitoring folder; lgtm!

File: support/index.adoc [hunk 2]
```diff
@@ -12,10 +12,20 @@ Red Hat offers cluster administrators tools for gathering data for your cluster,
 == Get support
 xref:../support/getting-support.adoc#getting-support[Get support]: Visit the Red Hat Customer Portal to review knowledge base articles, submit a support case, and review additional product documentation and resources.
 
+ifndef::openshift-rosa,openshift-dedicated[]
```
Comment (kcarmichael08) [inline support/index.adoc:15]: Just checking - should I be seeing this section in OSD and ROSA versions? I think the intent is for it to _not_ show, but I am seeing it there. I could be confused though.

- https://63305--docspreview.netlify.app/openshift-dedicated/latest/support/#support-overview-remote-health-monitoring
- https://63305--docspreview.netlify.app/openshift-rosa/latest/support/#support-overview-remote-health-monitoring

**Review by kcarmichael08 — COMMENTED** [review]
reviewed files in support folder, lgtm!

File: _topic_maps/_topic_map_osd.yml [hunk 3]
```diff
@@ -107,6 +104,87 @@ Topics:
...
+  - Name: Using Insights Operator
+    File: using-insights-operator
+# Not supported per Michael McNeill
+# - Name: Using remote health reporting in a restricted network
+#   File: remote-health-reporting-from-restricted-network
+# cannot list resource "secrets" in API group "" in the namespace "openshift-config"
+#  - Name: Importing simple content access entitlements with Insights Operator
+#    File: insights-operator-simple-access
+# must-gather not supported for customers, per Dustin Row, cannot create resource "namespaces" 
+# - Name: Gathering data about your cluster
```
Comment (ShaunaDiaz) [inline _topic_maps/_topic_map_osd.yml:143]: I see changes in this assembly and module below, but they aren't included. Should this be commented in?

File: support/gathering-cluster-data.adoc [hunk 4]
```diff
@@ -36,7 +36,9 @@ include::modules/gathering-data-specific-features.adoc[leveloffset=+2]
 
 == Additional resources
```
Comment (ShaunaDiaz) [inline support/gathering-cluster-data.adoc:38]: See comment on line 143 of the topic map. I don't see this assembly included.

**Review by ShaunaDiaz — COMMENTED** [review]
Lots of great work here! Kudos for keeping track! Just a few comments and questions. I do recommend checking the meta-data at the top of every moved file to make sure it lists the updated locations of this PR. I don't see tracked changes to that info.

File: support/index.adoc [hunk 5]
```diff
@@ -28,10 +38,12 @@ xref:../support/remote_health_monitoring/about-remote-health-monitoring.adoc#abo
 ** Identify potential cluster issues proactively.
 ** Provide a solution and preventive action in {cluster-manager-first}.
 
-You can xref:../support/remote_health_monitoring/showing-data-collected-by-remote-health-monitoring.adoc#showing-data-collected-by-remote-health-monitoring[Review telemetry information].
+You can xref:../support/remote_health_monitoring/showing-data-collected-by-remote-health-monitoring.adoc#showing-data-collected-by-remote-health-monitoring[review telemetry information].
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply support/index.adoc:41]: @ShaunaDiaz It seems at a quick glance that the Telemetry service is capped, but telemetry data, metrics, information (the stuff that the service gathers, presumably) is not. I think.

File: support/remote_health_monitoring/about-remote-health-monitoring.adoc [hunk 6]
```diff
@@ -58,42 +64,40 @@ ifndef::openshift-dedicated[]
...
 * Telemetry is installed and enabled by default. If you need to opt out of remote health reporting, see xref:../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#opting-out-remote-health-reporting[Opting out of remote health reporting].
-endif::[]
+endif::openshift-rosa,openshift-dedicated[]
 
 include::modules/insights-operator-about.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
+ifndef::openshift-rosa,openshift-dedicated[]
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply support/remote_health_monitoring/about-remote-health-monitoring.adoc:76]: @ShaunaDiaz Good catch. I missed that one, clearly!

File: _topic_maps/_topic_map_osd.yml (hunk 3 above)
Comment (mburke5678) [reply _topic_maps/_topic_map_osd.yml:143]: No everything in this assembly is done by RH for the customer.

File: support/index.adoc [hunk 7]
```diff
@@ -12,10 +12,20 @@ Red Hat offers cluster administrators tools for gathering data for your cluster,
...
 == Remote health monitoring issues
 xref:../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[Remote health monitoring issues]:
 {product-title} collects telemetry and configuration data about your cluster and reports it to Red Hat by using the Telemeter Client and the Insights Operator. Red Hat uses this data to understand and resolve issues in _connected cluster_. Similar to connected clusters, you can xref:../support/remote_health_monitoring/remote-health-reporting-from-restricted-network.adoc#remote-health-reporting-from-restricted-network[Use remote health monitoring in a restricted network]. {product-title} collects data and monitors health using the following:
+endif::openshift-rosa,openshift-dedicated[]
+
+// Removed sentence on restricted networks, not supported in ROSA/OSD
+ifdef::openshift-rosa,openshift-dedicated[]
+[id='support-overview-remote-health-monitoring']
+== Remote health monitoring issues
+xref:../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[Remote health monitoring issues]:
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply support/index.adoc:26]: I think that is allowed in the Index files.

File: support/gathering-cluster-data.adoc (hunk 4 above)
Comment (mburke5678) [reply support/gathering-cluster-data.adoc:38]: I learned after the fact that the assembly didn't need to be included. I think it OK to leave the `ifdef`s.

**Comment by mburke5678** [comment]
@ShaunaDiaz @kcarmichael08 Thanks for you quick and thorough reviews!!

**Pull request merged**